src/components/Integrations/DataStudio/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,63 @@
+
+import React, { Fragment } from 'react';
+import './styles/index.css'
+
+
+interface IFreeAccountProps {
+
+}
+
+const FreeAccount: React.SFC<IFreeAccountProps> = (props) => {
+    return <Fragment>
+        <section className="dp-gray-page p-t-54 p-b-54">
+            {/* <!--TODO refactor name of style class for this section--> */}
+            <div className="dp-calc-box" style={{width: "1250px"}}>
```

Comment:
We do not use inline styles 👀. How about adding a styled component for this particular fix?

Reply:
you could use <> instead <Fragment>. It's more simple